### PR TITLE
fixing some http codes

### PR DESCRIPTION
**Pull request by juanDuqueS**
what: fixed some http codes that was'nt apropiate

why: to give good answers 

Yaz-28: Review and fix the error codes returned by the API
Pd: sry for the gramatical errors in the name branch 🤕

### CODE REVIEW

File: back-end-yaz/src/server/controllers/ItemController.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ class ItemController {
             if (!Number(id)) {
                 util.setError(400, 'Please input a valid numeric value');
             } else if(updateItem){
-                util.setSuccess(200, `Item ${id} updated!`);
+                util.setSuccess(202, `Item ${id} updated!`);
```
Comment (alejandroVaron) [inline back-end-yaz/src/server/controllers/ItemController.js:43]: Are you sure its a 202 error?

**Review by alejandroVaron — APPROVED** [review]
nice!

File: back-end-yaz/src/server/controllers/ItemController.js (hunk 1 above)
Comment (juanDuqueS) [reply back-end-yaz/src/server/controllers/ItemController.js:43]: thanks for pointing it out! it is a 201 code

**Review by prometteo14 — APPROVED** [review]

File: back-end-yaz/src/server/controllers/ItemController.js [hunk 2]
```diff
@@ -76,11 +76,11 @@ class ItemController {
             if (!Number(id)) {
                 util.setError(400, 'Please provide a numeric value');
             } else if (req.body.user_type > 0) {
-                util.setError(400, `You do not have permission to do this!`);
+                util.setError(403, `You do not have permission to do this!`);
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/controllers/ItemController.js:79]: Please do the message a little bit specific.

File: back-end-yaz/src/server/controllers/Payment_itemsController.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ class Payment_itemsController {
             if(allPayment_items.length > 0){
                 util.setSuccess(200, 'Payment_items returned', allPayment_items)
             } else {
-                util.setSuccess(500, 'No payment_items found');
+                util.setSuccess(204, 'No payment_items found');
```
Comment (AngelicaAgudelo) [inline back-end-yaz/src/server/controllers/Payment_itemsController.js:14]: Please don't call the entity like we call it in the database.

**Review by AngelicaAgudelo — APPROVED** [review]
Ok.

**Pull request merged**